<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineDrawSelectionWidget</class>
 <widget class="QWidget" name="LineDrawSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Draw Line mode: Draw a line on the current frame only.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="drawingGroupBox">
     <property name="title">
      <string>Line Drawing</string>
     </property>
     <layout class="QVBoxLayout" name="drawingLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status: Ready to draw</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pointsLabel">
        <property name="text">
         <string>Points: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="startDrawingButton">
          <property name="text">
           <string>Start Drawing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="completeDrawingButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Complete Line</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionLayout">
        <item>
         <widget class="QPushButton" name="applyToCurrentFrameButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Apply to Current Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearLineButton">
          <property name="text">
           <string>Clear Line</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionsLabel">
     <property name="text">
      <string>Instructions:
1. Click "Start Drawing" to begin
2. Click in the video to add points to your line
3. Click "Complete Line" when finished drawing
4. Click "Apply to Current Frame" to add the line to the current frame</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
